layouts/_default/list.atom.xml

```diff
@@ -32,14 +32,211 @@
         {{- $copyright = $copyright | replace "&copy;" "©" -}}
         <rights>{{ $copyright | plainify }}</rights>
     {{- end }}
-    {{ with site.Author.name -}}
-        <author>
-            <name>{{ . }}</name>
-            {{ with site.Author.email }}
-                <email>{{ . }}</email>
-            {{ end -}}
-        </author>
-    {{- end }}
+    {{ $authors := "" }}
+    {{ if isset site "Author" }}
```

Comment:
I like using `with` instead of `isset` everywhere.

Then you can do `with` .. `else` and then nested `with` .. `else`  in that.

Reply:
I saw that it wouldn't work everywhere in case the node does not exist for some reason so I came to use `isset` instead.

Reply:
`with` would work the same way too.

Reply:
When I started off with Hugo, I learned that hard way to never use `isset`.. it's finicky with nil vs empty string or something like that.